Automattic/wp-calypso
# Me: Improve UI when photo and Gravatar profile are hidden

Fixes https://github.com/Automattic/wp-calypso/issues/63199

## Proposed Changes

### Clarifies language on the "hide Gravatar" toggle

Instead of saying "Hide Gravatar profile", say "Hide photo and Gravatar profile":

#### After

<img width="754" alt="image" src="https://user-images.githubusercontent.com/36432/176886851-22268aeb-2616-4f26-ad4c-9ba402a3a55c.png">

#### Before

<img width="739" alt="image" src="https://user-images.githubusercontent.com/36432/176682911-68fc7c61-e694-4987-bdcb-3954f91a09c3.png">

### Visually indicates when Gravatar is hidden

When the user's photo and Gravatar profile are hidden, the page appears as follows:

<img width="761" alt="image" src="https://user-images.githubusercontent.com/36432/176886954-9dfdaca5-cd89-4299-a90c-2c4d4b64643a.png">

<img width="853" alt="image" src="https://user-images.githubusercontent.com/36432/176886989-be09774a-c250-4e29-94a3-c919aa6facab.png">

When the user's photo and Gravatar are enabled, the page appears as follows:

<img width="775" alt="image" src="https://user-images.githubusercontent.com/36432/176887025-f03c7135-98ca-45a4-ae2a-012759efd9f2.png">

#### Testing Instructions

1. Navigate to `/me`.
2. Observe the page appears as documented above.
3. Toggle the "Hide photo and Gravatar profile" button.
4. Observe the changed display of the page.
